frontend/docs/recommended_practices.md

```diff
@@ -4,13 +4,108 @@ This document serves as reference for current practices and patterns that we wan
 
 We aim to maintain a reasonably consistent code base through these practices and welcome PRs to update and improve these recommendations.
 
-## Application
+## Code Styleguide
 
-### Unit Testing
+We have our coding styleguide baked into our [ESLint and Prettier rules](https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/static/package.json#L261) for TypeScript code an [Stylelint rules](https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/static/package.json#L556) for Sass code.
+
+Looking forward, we aim at setting more strict best practices using ESLint and Stylelint. For that, we are leveraging a project called [betterer][betterer], which keeps track of our errors when a given test is passed. You can run it using `npm run betterer` and it will break if you introduce any new eslint errors. If you want to ignore the new errors you can run `npm run betterer:update` to update the betterer.results file. We do not recommend adding or introducing new eslint errors.
```

Comment:
`[betterer][betterer]` doesn't link to anything, is that intentional?

Reply:
Oh, my bad. Adding it.